addons/hr_contract/models/resource.py

```diff
@@ -1,15 +1,18 @@
 # -*- coding:utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 from datetime import datetime
+from collections import defaultdict
 
-from odoo import fields, models
+from odoo import fields, models, api
 from odoo.osv.expression import AND
 
 
 class ResourceCalendar(models.Model):
     _inherit = ['resource.calendar']
 
-    contracts_count = fields.Integer("# Contracts using it", compute='_compute_contracts_count', groups="hr_contract.group_hr_contract_manager")
+    contracts_count = fields.Integer("# Contracts using it", compute='_compute_contracts_count', groups="hr_contract.group_hr_contract_manager", store=True)
```

Comment:
why did you store a compute field? :thinking:

Reply:
because we needed search filter for it

Reply:
may be you can use search attribute like this : `https://github.com/odoo/odoo/blob/18.0/addons/account/models/account_move_line.py#L412`